### ISSUE TITLE
Usage

### ISSUE THREAD

**Issue by SaidiAli** [Superbalist/laravel-google-cloud-storage#92]
Hello, I want to use your package but am having a hard time figuring out how to get google authentication. I do not want to use the json file but the i do not know how to find the details for the alternative. Will please provide a link for that or at least the steps i can take in my google console to find them.

There details, i do not know how to find them.
'key_file' => [
    'type' => env('GOOGLE_CLOUD_ACCOUNT_TYPE'),
    'private_key_id' => env('GOOGLE_CLOUD_PRIVATE_KEY_ID'),
    'private_key' => env('GOOGLE_CLOUD_PRIVATE_KEY'),
    'client_email' => env('GOOGLE_CLOUD_CLIENT_EMAIL'),
    'client_id' => env('GOOGLE_CLOUD_CLIENT_ID'),
    'auth_uri' => env('GOOGLE_CLOUD_AUTH_URI'),
    'token_uri' => env('GOOGLE_CLOUD_TOKEN_URI'),
    'auth_provider_x509_cert_url' => env('GOOGLE_CLOUD_AUTH_PROVIDER_CERT_URL'),
    'client_x509_cert_url' => env('GOOGLE_CLOUD_CLIENT_CERT_URL'),
],

**Comment by zawilliams** [comment]
@SaidiAli I just ran into this same thing.

I had to create a Service Account. I followed these instructions: https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys

Once you create the json key, it'll download it and you can copy and paste those values into your .env file.

**Comment by SaidiAli** [comment]
Hi @zawilliams thank you for that hack. I’ve just tried it out and it seems to work. But am failing to handle the private key. How can add it. I figured that I cannot add it to the env file, so I added it as it’s own file in the the storage path, then I gave that path as the value in the config file. But am getting an `openssl_sign(): supplied key param cannot be coerced into a private key`
How did you handle it?
